packages/cli/src/utils.ts

```diff
@@ -200,7 +203,19 @@ export interface Sample {
 
 export async function getTemplates(): Promise<Sample[]> {
```

Comment:
maybe the name should be like getSamples?

Reply:
It looks like the API name is related to CLI argument name "template-name", I think maybe @jayzhang could help to take a look and decide whether to rename it.